### PR TITLE
Fixed hideOverlay props for playable player

### PR DESCRIPTION
**Pull request by IlyaLytvynov**
Added ability to pas hideOverlay props directly in to Playable player

### CODE REVIEW

File: packages/wix-ui-core/src/components/video/Video.spec.tsx [hunk 1]
```diff
@@ -131,6 +131,23 @@ describe('Video', () => {
 
         expect(await driver.hasCover()).toBeTruthy();
       });
+
+      it('should not render', async () => {
```
Comment (kupriyanenko) [inline packages/wix-ui-core/src/components/video/Video.spec.tsx]: please make more specific name for the test case.

File: packages/wix-ui-core/src/components/video/Video.spec.tsx [hunk 2]
```diff
@@ -131,6 +131,23 @@ describe('Video', () => {
...
+                poster: IMAGE_SRC,
+              },
+            }}
+            hideOverlay
+            src={PLAYABLE_LINK}
+            id={VIDEO_ID}
+          />,
+        );
+
+        expect(await driver.hasCover()).not.toBeTruthy();
```
Comment (kupriyanenko) [inline packages/wix-ui-core/src/components/video/Video.spec.tsx]:
```suggestion
        expect(await driver.hasCover()).toBeFalsy();
```

File: packages/wix-ui-core/src/components/video/Video.spec.tsx (hunk 1 above)
Comment (IlyaLytvynov) [reply packages/wix-ui-core/src/components/video/Video.spec.tsx]: Done

File: packages/wix-ui-core/src/components/video/Video.spec.tsx (hunk 2 above)
Comment (IlyaLytvynov) [reply packages/wix-ui-core/src/components/video/Video.spec.tsx]: Fixed

**Comment by kupriyanenko** [comment]
@IlyaLytvynov let's squash the commits and then merge.

**Comment by IlyaLytvynov** [comment]
@shlomitc  commits squashed, I think it could be merged

**Pull request merged**